theano/gof/opt.py

```diff
@@ -2145,15 +2144,15 @@ def importer(node):
 
         def pruner(node):
             if node is not current_node and node.op == op:
-                try:
-                    q.remove(node)
-                except ValueError:
-                    pass
+                node.disavowed = True
```

Comment:
I made the same change to all pruners, I'll try deleting this one if it turns out to be unused.